### PR TITLE
Updating Models to allow for nested attributes

### PR DESCRIPTION
**Pull request by camidemery**

### CODE REVIEW

**Comment by leila-alderman** [comment]
It looks like you created this branch off of your last feature branch instead of creating it off of master. 

Could you please update this branch so that it shows only the relevant changes and the changes that you made in the last PR?

Now that your last PR has been merged, you can do this with
`git checkout master`
`git pull upstream master`
`git checkout models`
`git merge master`

File: app/models/facility.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 class Facility < ActiveRecord::Base
-    has_many :bill
+    has_many :bills
```
Comment (leila-alderman) [inline app/models/facility.rb:2]: [Style]
Please fix the spacing here so that the indentation is 2 spaces, not four.

File: app/models/item.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 class Item < ActiveRecord::Base
-  belongs_to :bill
+      belongs_to :bill
```
Comment (leila-alderman) [inline app/models/item.rb:2]: [Style]
Please fix the spacing here so that the indentation is 2 spaces, not six.

File: app/models/item.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
+    validates :name, :description, :price, presence: true
```
Comment (leila-alderman) [inline app/models/item.rb:3]: [Style]
Please fix the spacing here so that the indentation is 2 spaces, not four.

**Review by leila-alderman — CHANGES_REQUESTED** [review]
Looks good! Just some very minor spacing issues.

**Comment by camidemery** [comment]
Issues have been fixed on master already

**Pull request closed without merging**